src/components/collect-button.tsx

```diff
@@ -47,7 +47,7 @@ export class CollectButtonComponent extends BaseComponent<IProps, IState> {
           </svg>
         </div>
         <div className="collect-button-outline">
-          <div className={buttonClass} onClick={handleMouse} data-test="stored-mouse-class">
+          <div className={buttonClass} onMouseDown={handleMouse} data-test="stored-mouse-class">
             <div className={innerOutlineClass} data-test="inner-outline">
```

Comment:
onMouseUp would make the mouse action feel more similar to our other buttons and clickable objects where the action doesn't occur until the mouse up event fires.